docs/framework/install/net-framework-3-5-on-windows-8-plus.md

```diff
@@ -19,52 +19,56 @@ ms.author: "mairaw"
+## Install the .NET Framework 3.5 on Demand
+
+If an app requires the .NET Framework 3.5 but doesn't find that version enabled on your computer, it displays the following message box, either during installation or when you run the app for the first time. In the message box, choose **Install this feature** to enable the .NET Framework 3.5. This option requires an Internet connection.  
   
- ![Dialog box for 3.5 install on Windows 8](../../../docs/framework/deployment/media/installdialog.png "installdialog")  
+![Dialog box for 3.5 install on Windows 8](../../../docs/framework/deployment/media/installdialog.png "installdialog")  
   
 <a name="ControlPanel"></a>   
-## Enable the .NET Framework 3.5 in Control Panel  
- You can enable the .NET Framework 3.5 yourself through Control Panel. This option requires an Internet connection.  
+## Enable the .NET Framework 3.5 in Control Panel
+
+You can enable the .NET Framework 3.5 through Control Panel. This option requires an Internet connection.  
   
-1.  Press the Windows key ![Windows logo](../../../docs/framework/get-started/media/windowskeyboardlogo.png "Windowskeyboardlogo") on your keyboard, type Windows Features, and press Enter. This brings up the **Turn Windows features on or off** dialog box. Alternately, open Control Panel, click on the Programs items, and then click on "Turn Windows features on or off" under Programs and Features.  
+1. Press the Windows key ![Windows logo](../../../docs/framework/get-started/media/windowskeyboardlogo.png "Windowskeyboardlogo") on your keyboard. Type "Windows Features" and press Enter. This brings up the **Turn Windows features on or off** dialog box. Alternately, open Control Panel, click on the **Programs** items, and then select **Turn Windows features on or off** under **Programs and Features**.  
   
-2.  Select the **.NET Framework 3.5 (includes .NET 2.0 and 3.0)** check box, press OK, and reboot your computer if prompted.  
+2. Select the **.NET Framework 3.5 (includes .NET 2.0 and 3.0)** check box, select **OK**, and reboot your computer if prompted.  
   
- You do not need to select the child items for Windows Communication Foundation (WCF) HTTP activation unless you are a developer who requires WCF script and handler mapping functionality.  
+You don't need to select the child items for **Windows Communication Foundation (WCF) HTTP activation** unless you're a developer who requires WCF script and handler mapping functionality.  
   
- The following video shows how to do this:  
+The following video shows how to do this:  
   
- ![Installing the .NET Framework on Windows 8 or 8.1](../../../docs/framework/get-started/media/clr-net35-win8.png "CLR_NET35_Win8")  
+![Installing the .NET Framework on Windows 8 or 8.1](../../../docs/framework/get-started/media/clr-net35-win8.png "CLR_NET35_Win8")  
```

Comment:
Yes and given the instruction in Step 2, the entire statement/link can be removed. It's *not complicated.*